client/src/pages/Materials/index.scss

```
@@ -98,7 +109,7 @@
     }
 
     &__actions {
-        min-width: 110px;
+        display: flex;
```

J'aurais plutôt tendance à utiliser: 

```suggestion
        width: 1%;
        text-align: right;
        white-space: nowrap;
```

Même si c'est un "hack", ça permet à la colonne de prendre toujours le minimum de place possible (uniquement pour que ses boutons rentrent) et pas de se retrouver avec les boutons au milieu s'il n'y a que deux colonnes et que la première est petite aussi.

---

À virer en fait, ça sera fait dans la PR #338 finalement.